tools/sigh.ts

```diff
@@ -169,14 +169,14 @@ function check(): boolean {
   // itself is quite verbose). The regex extracts the version from a line that looks like:
   //   emcc (Emscripten gcc/clang-like replacement) 1.38.38 (4965260 Tue Jul 9 13:29:04 2019 ...
   const emsdkResult = saneSpawnWithOutput('npx', ['emsdk-run', 'em++', '--version']);
-  const match = emsdkResult.stdout.match(/\nemcc [^0-9.]+ (X[0-9.]+) /);
+  const match = emsdkResult.stdout.match(/\s[0-9]+.[0-9]+.[0-9]+\s/);
```

Comment:
No, this was deliberate. The output is multi-line and could include other dotted numeric values.

The leftover 'X' was a mistake, however (forcing local failure to debug).

Reply:
Ah. Sorry, I was trying to get the build to work. I'm glad you were able to diagnose the root issue.